docs: first set of eCHO video links added (#20480)
cilium-team/cilium: Documentation/bpf.rst

        typedef unsigned int u32;
        [...]

> .. admonition:: Video

Reviewer: Please add a blank line.

```suggestion

.. admonition:: Video
```

Author: Implemented the fix. Thank you.

  :class: attention

  To learn more about bpftool, check out `eCHO episode 11: Exploring 

>   bpftool <https://www.youtube.com/watch?v=1EOLh3zzWP4&t=650s>`__ with Quentin Monnet, a maintainer of bpftool.

Reviewer: ```suggestion
  bpftool <https://www.youtube.com/watch?v=1EOLh3zzWP4&t=650s>`__ with Quentin Monnet, maintainer of bpftool.
```

        typedef unsigned int u32;
        [...]

To learn more about bpftool, check out eBPF & Cilium Office Hours `episode 11: Exploring 

> bpftool <https://www.youtube.com/watch?v=1EOLh3zzWP4&t=650s>`__ with Quentin Monnet, a maintainer of bpftool.

Reviewer: ```suggestion
To learn more about bpftool, check out `eCHO Episode 11: Exploring 
bpftool <https://www.youtube.com/watch?v=1EOLh3zzWP4&t=650s>`__ with Quentin Monnet, maintainer of bpftool.
```

Reviewer 2: Not sure if you meant to resolve this conversation without taking the suggestion? I think it's worth taking to make it consistent

mailing list (http://vger.kernel.org/vger-lists.html#xdp-newbies), which explain
various parts of XDP and BPF:

> 1. May 2017,

Reviewer: Please keep the reverse order. If we ever change them we want to do it consistently on the whole page (i.e. other sections too), and this is beyond the scope of this PR.

Looking at the commit history, it looks like you reverted this change, but somehow reintroduced it (inadvertently?) in the last commit.

Author: Thank you will do so.

Author: Done.

The following (incomplete) list includes talks and conference papers
related to BPF and XDP:

46. July 2021,
     eBPF & Cilium Office Hours episode 13: XDP Hands-on Tutorial, with Liz Rice,
     https://www.youtube.com/watch?v=YUI78vC4qSQ&t=300s

45. June 2021,
     eBPF & Cilium Office Hours episode 9: XDP and Load Balancing, 

>      with Daniel Borkmann, https://www.youtube.com/watch?v=OIyPm6K4ooY&t=308s

Reviewer: Consistency

```suggestion
     with Daniel Borkmann,
     https://www.youtube.com/watch?v=OIyPm6K4ooY&t=308s
```

Author: Thank you.
